tests/lib/prepare-project.sh

```diff
@@ -119,18 +121,15 @@ if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
     quiet apt-get install -y --force-yes apparmor libapparmor1 seccomp libseccomp2 systemd cgroup-lite util-linux
 fi
 
-quiet apt-get purge -y snapd
+distro_purge_package snapd
```

Comment:
My point that abstract identifiers are more harm/confusion than benefit. Maybe on Debian something is in one package but in another distribution you need two packages (for whatever reason, maybe the base is different). Trying to cram this into a ever-growing table that one must look up each time a new test is written feels wrong to me. I'd much rather implement a "install_package" function that installs packages, as they are spelled out in the argument list, simply ignoring packages for foreign distributions (hence the `${ID}:${package}` syntax). The case with gentoo is that packages have widely different, long and path-like names. I really think this is the wrong abstraction.

Let test developers just install what they want. Alternatively my idea can be converted to:

```
maybe_apt install some packages
maybe_yum install other library--devel too
maybe_zypper ...
maybe_emerge
```

Where all the `maybe_` functions are a no-op unless the given packaging tool exists. (Though I still prefer the thing to be spelled on one line as in my original proposal)